poetry.lock

```diff
@@ -8,17 +8,17 @@ python-versions = "*"
 
 [[package]]
 name = "alembic"
-version = "1.5.6"
+version = "1.5.7"
```

Comment:
[alembic 1.5.7](https://alembic.sqlalchemy.org/en/latest/changelog.html#change-1.5.7) is a bug fix